lighthouse/classes/messages/sequencescape_messages.py

```diff
@@ -1,6 +1,73 @@
+from flask import current_app as app
 from lighthouse.messages.message import Message
+from typing import Dict
+import requests
+import logging
 
+logger = logging.getLogger(__name__)
 
-class Sequencescapemessage(Message):
+
+class SequencescapeMessage(Message):
     def __init__(self):
-        ...
+        self._barcode = None
+        self._contents = {}
+
+    def set_barcode(self, barcode):
+        self._barcode = barcode
+
+    def set_well_sample(self, location, sample_info):
+        self._contents[location] = sample_info
+
+    def render(self) -> Message:
+        message_content = self.construct_sequencescape_message(
+            barcode=self._barcode,
+            purpose_uuid=app.config["SS_UUID_PLATE_PURPOSE_CHERRYPICKED"],
+            study_uuid=app.config["SS_UUID_STUDY_CHERRYPICKED"],
+            wells=self._contents,
+            events=[],
+        )
+
+        return Message(message_content)
+
+    def construct_sequencescape_message(self, barcode, purpose_uuid, study_uuid, wells, events):
+        body = {
+            "barcode": barcode,
+            "purpose_uuid": purpose_uuid,
+            "study_uuid": study_uuid,
+            "wells": wells,
+            "events": events,
+        }
+
+        return {"data": {"type": "plates", "attributes": body}}
+
+    def send_to_ss(self):
+        message = self.render()
+        return self._send_to_ss(
+            ss_url=f"http://{app.config['SS_HOST']}/api/v2/heron/plates",
```

Comment:
I think we should move this URL to config.

Reply:
ok, I will move it

Reply:
Done